### PR TITLE
newsletter 8/24

### PR DESCRIPTION
**Pull request by vs77bb**
@owocki pls review. feel free to make any edits inline!

### CODE REVIEW

File: app/retail/emails.py [hunk 1]
```diff
@@ -551,40 +553,40 @@ def render_new_bounty_roundup(to_email):
...
+            'what': 'Built out the Gitcoin Requests highlighted above!',
+            'link': 'https://gitcoin.co/issue/gitcoinco/web/2036/982',
             'link_copy': 'View more',
         },
         {
-            'who': 'tcrowe',
+            'who': 'evgeniuz',
             'who_link': True,
-            'what': 'Was one worker picked out of 8, and did a great job on the Bounties website.',
+            'what': 'Helped MetaMask to remember recent RPC URL's.',
```
[the remark this answers is not in the document]
Comment (owocki) [reply app/retail/emails.py]: syntax error.. you need to escape the `'` like soee5aa2d772d2127779fb784967ca3bb3a7a65d0a

File: app/retail/emails.py [hunk 2]
```diff
@@ -513,36 +513,38 @@ def render_start_work_applicant_expired(interest, bounty):
...
-Are you interested in helping to build test suites across the Ethereum ecosystem?
-We're working with the Ethereum Foundation to find the best and brightest developers to get involved directly on Ethereum's
-codebase and would love to hear from you. If you're interested, join #focus-dev-testing <a href="https://gitcoin.co/slack">on Gitcoin Slack</a>
-and/or respond to this e-mail with a bit more about yourself / your development background. Bonus points if you have an active Gitcoin profile!
+This week, we launched <a href="https://gitcoin.co/requests/">Gitcoin Requests</a>. Gitcoin Requests allows funders and developers to request bounties on
+specific Github issues. We hope that maintainers and developers alike use this feature to let us know what else they'd
+like to have funded so we can continue building at break-neck speed! <a href="https://gitcoin.co/requests/">Checkout Gitcoin Requests here</a>.
 </p>
-
+To this end, we have recently received $60K grant from the Ethereum Community Fund to post bounties across a variety of Web 3
```
Comment (owocki) [inline app/retail/emails.py]: missing p tag 2de1e9784c790f5b5f8428cef07f4ecc0fa9a315

File: app/retail/emails.py [hunk 3]
```diff
@@ -551,40 +553,40 @@ def render_new_bounty_roundup(to_email):
...
             'who_link': True,
-            'what': 'Completed his first bounty on the Ethereum Utils repo!',
-            'link': 'https://gitcoin.co/issue/ethereum/eth-utils/117/964',
+            'what': 'Built out the Gitcoin Requests highlighted above!',
+            'link': 'https://gitcoin.co/issue/gitcoinco/web/2036/982',
             'link_copy': 'View more',
         },
         {
-            'who': 'tcrowe',
+            'who': 'evgeniuz',
```
Comment (owocki) [inline app/retail/emails.py]: @mbeacom the svgs we serve dont seem to render in gmail.  any good workarounds youre aware of here? http://bits.owocki.com/a0729b318442/Screen%20Shot%202018-08-24%20at%2010.31.27%20AM.png

Comment (mbeacom) [reply app/retail/emails.py]: @owocki Which SVGs seem to be not rendering?

Comment (owocki) [reply app/retail/emails.py]: look at the screenshot in my comment above or in triage channel

see https://gitcoin.co/_administration/email/roundup if you want to verify that its a SVG specific issue

**Pull request merged**